backend/testing.py

```diff
@@ -439,7 +439,7 @@ def get_sample_encoded_art_image(filepath="") -> str:
     if filepath == "":
         # select random image if not supplied with a path
         path = "./assets/sample-artworks"
-        image_paths = os.listdir(path)
+        image_paths = [img for img in os.listdir(path) if img.endswith(".jpg")]
```

Comment:
Is this to address what you were mentioning about `.DS_Store`?

Reply:
Yeah and any other files that aren't images. Long term if we add other image types, I'll replace the `endswith` with a regex `/png|jpe*g|webp/` to cover all our img suffixes